miio/cooker_profiles.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
Maybe `miio/data/cooker_profiles.json` to keep it from polluting the main source dir?

Reply:
To add, the setup.py and/or manifest file will require also an update, otherwise the json file won't be added when generating a package.